index.html

```diff
@@ -42,5 +42,175 @@ <h1 class="name"> Hey there <br>
           <img src="images/Disabled.svg" alt="drop-down">
         </div>
     </section>
+    <div class="line-break">
+        <h2 class="works">My Recent <br> Works</h2>
+        </div>
+      <section class="section-placeholder">
```

Comment:
great work so far 👍🏼  , I would just suggest that you put this title inside the section as it is part of it always wrap all related content in one section also using a descriptive class name is a good practice works section refers better to the content of this section than section-placeholder